lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -882,8 +883,7 @@ def test_cache_map_get_info_cmap(
         [
             ("", CacheType.FILE, io.BytesIO(b"tarball_as_a_byte_stream")),
             (None, CacheType.FILE, io.BytesIO(b"tarball_as_a_byte_stream")),
-            ("f1.json", CacheType.FILE, io.BytesIO(b"tarball_as_a_byte_stream")),
-            ("subdir1/subdir12", CacheType.DIRECTORY, None),
```

Comment:
Do we have a test for what happens when someone calls `Tarball.filestream()` for something other than a `CacheType.FILE`?

Reply:
Ah; yeah, I'd meant to backfill this, but dropped that ball. 😦